api/model.go

```diff
@@ -124,6 +124,11 @@ type SnapshotCR struct {
 type BackupTarget struct {
 	client.Resource
 	engineapi.BackupTarget
+	SyncBackupTarget bool `json:"syncBackupTarget"`
```

Comment:
This is more like a workaround to hijack to the `BackupTarget` resource. Or you want to have a Sync resource for this purpose instead?

for example:
```go
type SyncBackupResource struct {
    SyncAllBackupTargets bool
    SyncAllBackupVolumes bool
}
```

Reply:
Used the struct `SyncBackupResource` instead. Thanks.